code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1444,7 +1444,7 @@
 /mob/living/carbon/human/has_eyes()
 	if(internal_organs_by_name["eyes"])
 		var/datum/organ/internal/eyes = internal_organs_by_name["eyes"]
-		if(eyes && istype(eyes) && !eyes.status & ORGAN_CUT_AWAY)
+		if(eyes && istype(eyes) && !(eyes.status & ORGAN_CUT_AWAY))
```

Comment:
I'm pretty sure that if `ORGAN_CUT_AWAY` happened to be anything other than 1, no humans would have been considered to have eyes before. Also, if their eyes had *any* `status` they would have been considered not to have eyes, instead of just `ORGAN_CUT_AWAY`.